stable/spartakus/templates/deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "fullname" . }}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+    spec:
+      containers:
+        - name: {{ template "fullname" . }}
+          image: "{{ .Values.image }}"
+          imagePullPolicy: {{ default "" .Values.imagePullPolicy | quote }}
+          args:
+            - "volunteer"
+            - "--cluster-id={{ uuidv4 | default .Values.uuid }}"
+          resources:
+            {{ include "toYaml" .Values.resources | indent 12 }}
```

Comment:
I think this needs to be:
```
          resources:
{{ toYaml .Values.resources | indent 12 }}
```

Note with the built-in function, the first line will also be indented so that line needs to not be indented.